### PR TITLE
User identity added to new client

### PR DESCRIPTION
**Pull request by chateaux**
Added the ability to add a user_id to a new client.

### CODE REVIEW

**Comment by chateaux** [comment]
Big fix

File: src/DoctrineGui/Controller/DoctrineGuiController.php [hunk 1]
```diff
@@ -277,9 +277,13 @@ public function clientManageAction()
             {
                 $clientObject = $this->clientService->find($client_id);
                 $this->clientForm->bind($clientObject);
+                $user_id = $clientObject->getUser()->getId();
+            } else {
+                $clientObject = new Client();
+                $user_id = '';
```
Comment (TomHAnderson) [inline src/DoctrineGui/Controller/DoctrineGuiController.php]: $user_id should be set to null

File: src/DoctrineGui/Controller/DoctrineGuiController.php [hunk 2]
```diff
@@ -277,9 +277,13 @@ public function clientManageAction()
...
             }
 
-            return new ViewModel( array( 'form' => $this->clientForm , 'client_id' => $client_id) );
+            return new ViewModel( array( 'form' => $this->clientForm , 'client_id' => $client_id , 'user_id' => $user_id) );
```
Comment (TomHAnderson) [inline src/DoctrineGui/Controller/DoctrineGuiController.php]: Break this command into multiple lines

```
return new ViewModel(array(
    'form' => $this->clientForm,
    'client_id => $client_id,
    ...
));
```

File: src/DoctrineGui/Controller/DoctrineGuiController.php [hunk 3]
```diff
@@ -361,8 +366,6 @@ public function clientManageAction()
             $clientObject->addScope($scopeObject);
         }
 
-        $clientObject->setUser();  //@TODO Add user
-
         if ($new_password != '')
```
Comment (TomHAnderson) [inline src/DoctrineGui/Controller/DoctrineGuiController.php]: Use camel case for non-foreign key variables.  Should be $newPassword.  Also no need to check for empty string, use `if ($newPassword) {`

File: src/DoctrineGui/Form/ClientFieldset.php [hunk 4]
```diff
@@ -49,6 +49,20 @@ public function init()
         );
 
         $this->add([
+                'name'          => 'user',
```
Comment (TomHAnderson) [inline src/DoctrineGui/Form/ClientFieldset.php:52]: Either use space padded `=>` to align all elements or don't use space padding at all.  I prefer no space padding.

File: src/DoctrineGui/Controller/DoctrineGuiController.php (hunk 3 above)
Comment (chateaux) [inline src/DoctrineGui/Controller/DoctrineGuiController.php]: Ok - will do.

File: src/DoctrineGui/Controller/DoctrineGuiController.php (hunk 2 above)
Comment (chateaux) [inline src/DoctrineGui/Controller/DoctrineGuiController.php]: Noted.

File: src/DoctrineGui/Form/ClientFieldset.php (hunk 4 above)
Comment (chateaux) [inline src/DoctrineGui/Form/ClientFieldset.php:52]: Do you mean like: 'name' => 'user', (i.e. a single space?) and not using padding would be: 'name'=>'user' ?

Comment (TomHAnderson) [inline src/DoctrineGui/Form/ClientFieldset.php:52]: Yes, like:

```
'key' => 'value',
'otherkey => 'othervalue',
```

The way to do this is not defined in PSR1 or PSR2.  It is defined in Symfony that you should align them like

```
'key'      => 'value',
'otherkey' => 'othervalue',
```

so it's your choice of style, I just want it always applied or never applied, not a mix.

**Comment by chateaux** [comment]
Hi Tom, was this pulled across?

**Pull request merged**